redis/src/test/java/org/apache/calcite/adapter/redis/RedisDataITCaseBase.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.adapter.redis;
+
+import org.junit.After;
+import org.junit.Before;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.JedisPoolConfig;
+
+/**
+ * RedisDataTypeTest.
+ */
+public class RedisDataITCaseBase extends RedisITCaseBase {
+  private JedisPool pool;
+
+  @Before
+  public void setUp() {
+    try {
+      JedisPoolConfig jedisPoolConfig = new JedisPoolConfig();
+      jedisPoolConfig.setMaxTotal(10);
+      pool = new JedisPool(jedisPoolConfig, HOST, PORT);
+    } catch (Exception e) {
+      throw e;
+    }
+  }
+
+  public void makeData() {
+    try (Jedis jedis = pool.getResource()) {
+      jedis.del("raw_01");
+      jedis.del("raw_02");
+      jedis.del("raw_03");
+      jedis.del("raw_04");
+      jedis.del("raw_05");
+      jedis.del("json_01");
+      jedis.del("json_02");
+      jedis.del("json_03");
+      jedis.del("json_04");
+      jedis.del("json_05");
+      jedis.del("csv_01");
+      jedis.del("csv_02");
+      jedis.del("csv_03");
+      jedis.del("csv_04");
+      jedis.del("csv_05");
```

Comment:
Note that `del` can take multiple keys as parameters, so this would likely be better written as one statement (or perhaps one for each data format.

Reply:
The same goes for `lpush` and `sadd` below. That is, you can write for example
```java
jedis.lpush("raw_02", "book1", "book2");
```